packages/cf/cf.0.5.0/opam

```diff
@@ -10,7 +10,7 @@ homepage: "https://github.com/mirage/ocaml-cf"
 doc: "https://mirage.github.io/ocaml-cf/"
 bug-reports: "https://github.com/mirage/ocaml-cf/issues"
 depends: [
-  "dune" {>= "2.9"}
+  "dune" {>= "3.0"}
```

Comment:
do you remember what was the error message?

Reply:
yes, see my previous comment above: https://github.com/ocaml/opam-repository/pull/21090#discussion_r841961613